Composer/packages/client/src/pages/design/DebugPanel/TabExtensions/WebChatLog/WebChatInspectorPane.tsx

```diff
@@ -0,0 +1,152 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/** @jsx jsx */
+import { JsonEditor } from '@bfc/code-editor';
+import { css, jsx } from '@emotion/core';
+import { NeutralColors } from '@uifabric/fluent-theme';
+import { Pivot, PivotItem } from 'office-ui-fabric-react/lib/Pivot';
+import { useCallback } from 'react';
+import { Resizable } from 're-resizable';
+
+import { WebChatInspectionData } from '../../../../../recoilModel/types';
+
+type ActivityType = 'conversationUpdate' | 'message' | 'trace';
+
+const editorStyles = css`
+  border: none;
+`;
+
+const pivotStyles = css`
+  margin-bottom: 8px;
+`;
+
+type WebChatInspectorPaneProps = {
+  inspectionData?: WebChatInspectionData;
+  setInspectionData: (data: WebChatInspectionData) => void;
+};
+
+const getUserFriendlyActivityType = (type: ActivityType | undefined) => {
+  switch (type) {
+    case 'conversationUpdate':
+      return 'Conversation update';
```

Comment:
nit: Conversation Update?

Reply:
this capitalization is represented in the designs